<launch>

  <!-- start the node "move_camera_b" which provides the services "start_move_camera" and "stop_move_camera" -->
  <node name="move_camera_b" pkg="motion_control" type="move_camera_b" respawn="true" output="screen">
  </node>

  <!-- start the node "rough_localizer_b" which provides the services "start_rough_localizer" and "stop_rough_localizer" -->
  <node name="rough_localizer_b" pkg="object_localizer" type="rough_localizer_b" respawn="true" output="screen">
  </node>

  <!-- start the node "box_segmenter_b" which provides the services "start_box_segmenter" and "stop_box_segmenter" -->
  <node name="box_segmenter_b" pkg="object_localizer" type="box_segmenter_b" respawn="true" output="screen">
  </node>

  <!-- start the node "scan_planner_b" which provides the service "start_scan_planner" -->
  <node name="scan_planner_b" pkg="object_localizer" type="scan_planner_b" respawn="true" output="screen">
    <param name="scan_plan_file" type="str" value="scan_plan.cfg"/>
  </node>

  <!-- start the node "do_scan_b" which provides the service "start_do_scan" -->
  <node name="do_scan_b_2" pkg="motion_control" type="do_scan_b_2" respawn="true" output="screen">
    <param name="scan_plan_file" type="str" value="scan_plan.cfg"/>
    <param name="scan_idx" type="int" value="1"/>
  </node>

  <!-- start the node "profile_merger" which provides the service "start_profile_merger" and "stop_profile_merger" -->
  <node name="profile_merger_c" pkg="object_localizer" type="profile_merger_c" respawn="true" output="screen">
  </node>

  <!-- start the node "rivet_localizer_c" which provides the service "start_rivet_localizer" -->
  <node name="rivet_localizer_c" pkg="object_localizer" type="rivet_localizer_c" respawn="true" output="screen">
  </node>

  <!-- start the node "point_rivet_c" which provides the service "start_point_rivet" -->
  <node name="point_rivet_c" pkg="motion_control" type="point_rivet_c" respawn="true" output="screen">
    <param name="XDK" type="int" value="-1"/>
    <param name="ip_address" type="str" value="192.168.2.183"/>
  </node>

</launch>
